.gitignore

```diff
@@ -10,4 +10,5 @@ packages/docs/pages/dev.tsx
 .idea
 .DS_Store
 .parcel-cache
-.tsbuildinfo
\ No newline at end of file
+.tsbuildinfo
+.env*.local
```

Comment:
This is NextJS's recommendation to only `.gitignore` the `.local` postfix files.